torch/_subclasses/fake_tensor.py

```diff
@@ -643,6 +674,8 @@ def run_fallback_kernel(func, args, kwargs, orig_not_implemented_exception):
         def to_real_tensor(e):
             if isinstance(e, FakeTensor):
                 out = torch.zeros_like(e, device=e.fake_device)
```

Comment:
so I guess `zeros_like` doesn't propagate the `coalesced`-ness of the input?

Reply:
No, because you always end up with nnz=0 for zeros :)